src/__main__.py

```diff
@@ -0,0 +1,20 @@
+"""
+The first module to do something
+"""
+
+import sys
+
+
+def echo(phrase: str) -> None:
```

Comment:
type annotations add some help and "safety" to the codebase.

this function `echo`  takes in an argument `phrase` that is a type of `string` and will return `None`

python still lets you shoot yourself in the foot if you want, but the type hints will pop up in your editor to try and keep you from doing so

Reply:
So what you're telling me.... is I need to watch some youTube 101 courses on Python to understand what you mean :D